Client/ClientGameStateMachine.cpp

```diff
@@ -0,0 +1,63 @@
+#include "ClientGameStateMachine.h"
+#include "State.h"
+#include "StateTransition.h"
+#include "GameStateInactive.h"
+#include "GameStateLoading.h"
+#include "GameStateMenu.h"
+#include "GameStateInGameplay.h"
+using namespace NCL;
+using namespace CSC8503;
+ClientGameStateMachine::ClientGameStateMachine() : StateMachine()
+{
+	gameWorld = new GameWorld();
+	renderer = new GameTechRenderer(*gameWorld);
```

Comment:
Decide if you want the renderer to be made BEFORE reaching this state or at this point